Code/GraphMol/FMCS/Wrap/rdFMCS.cpp

```diff
@@ -7,6 +7,10 @@
 //  which is included in the file license.txt, found at the root
 //  of the RDKit source tree.
 //
+#ifdef _WIN32
```

Comment:
I think you can replace this definition with the one that's already in the RDKit:
https://github.com/rdkit/rdkit/blob/master/Code/RDBoost/Wrap.h#L35